tox.ini

```diff
@@ -30,7 +33,26 @@ commands=
     poetry run yamllint --version
     poetry run yamllint -s .
 
-[testenv:py3]
-commands =
-    {[testenv]commands}
+[testenv:py36]
+deps = poetry >= 1.0.5
+       mock
+       ansible
+passenv = HOME
+skip_install = true
+commands=
```

Comment:
Wondering if a few of these settings (`passenv`, `skip_install`, and `commands`) need to be here, or if they'll automatically be inherited because they're already in the `testenv` section.